multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -53,7 +53,7 @@ def __init__(
             self._label_generator = label_generator
 
         # Scaler used for numerical labels
-        numerical_label_preprocessing = OmegaConf.select(config, "label.numerical_label_preprocessing")
+        numerical_label_preprocessing = OmegaConf.select(self._config.data, "label.numerical_label_preprocessing")
```

Comment:
If we pass the model config to the ner label encoder initialization, there would be no need to change the config in this file.